.github/workflows/manylinux_wheels.yml

```diff
@@ -42,14 +42,14 @@ jobs:
       CIBW_BUILD:  ${{ matrix.cibw_build }}
       CIBW_ARCHS: ${{ matrix.cibw_archs }}
       CIBW_BEFORE_ALL_LINUX: >
-        source .ci/ubuntu_ci.sh &&
+        mv `pwd`/kivy_build $HOME &&
         yum install -y epel-release &&
-        build_and_install_linux_kivy_sys_deps
+        yum -y install autoconf automake cmake gcc gcc-c++ git make pkgconfig zlib-devel portmidi portmidi-devel xorg-x11-server-devel mesa-libEGL-devel mtdev-devel mesa-libEGL freetype freetype-devel openjpeg openjpeg-devel libpng libpng-devel libtiff libtiff-devel libwebp libwebp-devel dbus-devel dbus ibus-devel ibus libsamplerate-devel libsamplerate libudev-devel libmodplug-devel libmodplug libvorbis-devel libvorbis flac-devel flac libjpeg-turbo-devel libjpeg-turbo wget
```

Comment:
Looks good, but this should be in its own function in `.ci/ubuntu_ci.sh` so you can call it here as well as in https://github.com/kivy/kivy/blob/master/.ci/ubuntu_ci.sh#L166. The idea is to have minimal actual code in these actions so it's easier to understand and in case we want to migrate from github actions in the future.